docs/vscode-extension/EXTRACT_STYLING.md

```diff
@@ -0,0 +1,63 @@
+# Otter Extract Styling to o3r.variable
+
+Add commands to extract scss properties to Otter variable
```

Comment:
to Otter variables to extract ... into (not sure for into @sdo-1A can you confirm?) Otter variableS